packages/charts/src/components/ColumnChart/ColumnChart.tsx

```diff
@@ -235,7 +242,7 @@ const ColumnChart: FC<ColumnChartProps> = forwardRef((props: ColumnChartProps, r
             // eslint-disable-next-line @typescript-eslint/ban-ts-comment
             // @ts-ignore
             label={{ value: chartConfig.secondYAxis.name, offset: 2, angle: +90, position: 'center' }}
-            orientation="right"
+            orientation={useIsRTL() === true ? 'left' : 'right'}
```

Comment:
please don't call hooks inside props. extract it to `const isRTL = useIsRTL()` somewhere before the return and just use the value here.